test/mason/mason-modify/masonModifyTest.chpl

```diff
@@ -20,25 +17,19 @@ proc main(args: [] string) {
   if exists(basename) then rmTree(basename);
 
   // Create new mason project directory
-  mkdir(basename);
+  mkdir('tmp/' + basename, parents=true);
 
   // Replace manifest file
   const tomlFile = basename + '.toml';
-  const manifestFile = basename + '/Mason.toml';
+  const manifestFile = 'tmp/' + basename + '/Mason.toml';
   FileSystem.copyFile(tomlFile, manifestFile);
 
-  // here.chdir is not sufficient
-  const oldPWD = getEnv('PWD');
-  const newPWD = '/'.join(oldPWD, basename);
-  setenv("PWD", newPWD.c_str(), 1);
+  here.chdir('tmp/' + basename);
 
   // Add mason dependency
   var modArgs: [0..#args.size-2] string = args[2..];
   masonModify(modArgs);;
 
   // Print manifest for diff against .good file
-  showToml(manifestFile);
-
-  // Cleanup project
-  rmTree(basename);
+  showToml('Mason.toml');
```

Comment:
Using absolute paths can make this line more maintainable, but it is a small enough test that I am not very familiar with. So, you can ignore :)

Reply:
The `masonModify` command above relies on being in the local path so making this an abs path doesn't save any effort here, unfortunately.